core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -913,6 +913,10 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-aws-qe
     namespace: ci
+  - cluster_groups:
+    - non_app_ci
+    name: cluster-secrets-azure-qe
```

Comment:
I think in https://github.com/openshift/release/pull/68471, you are using `cluster_profile: azure-oadp-qe`.  While in here, it's for `azure-qe`.  Is it expected ?